src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3150,6 +3152,14 @@ private VisitArgumentResult VisitArgumentEvaluate(BoundExpression argument, RefK
             Debug.Assert(!this.IsConditionalState);
         }
 
+        private void CheckDisallowedNullAssignment(TypeWithState state, TypeWithAnnotations type, FlowAnalysisAnnotations annotations, Location location)
```

Comment:
Does it make sense to make this a local function?

Reply:
I expect to use the method for checking on scenario when assigning to fields and properties too.

---
In reply to: [292627658](https://github.com/dotnet/roslyn/pull/36263#discussion_r292627658) [](ancestors = 292627658)